apps/condo/domains/ticket/constants.js

```diff
@@ -55,6 +55,25 @@ const MAX_COMMENT_LENGTH = 700
 
 const DEFAULT_DEFERRED_DAYS = 30
 
+const TICKET_OTHER_SOURCE_ID = '7da1e3be-06ba-4c9e-bba6-f97f278ac6e4'
+
+/**
+ * @example
+ * updateTickets - Query name in Ticket.updateMany. Usage in Ticket.test.js
+ * updateTicketsForReassignmentEmployee - Query name in query/Ticket.graphql. Usage in DeleteEmployeeButtonWithReassignmentModal.jsx
+ */
+const DISABLE_PUSH_NOTIFICATION_FOR_OPERATIONS = [
+    'updateTickets',
```

Comment:
We don't have this query
```suggestion
```

Reply:
I need this query for tests. Frontend queries cannot be used in tests

Reply:
> I need this query for tests. Frontend queries cannot be used in tests

You can write the query you need for tests. You should to make sure that "updateTicketsForReassignmentEmployee" works correctly